syntax = "proto3";

service NumberStreamService {
  rpc GetNumbers (GetNumbersRequest) returns (stream GetNumbersResponse) {}
}

message GetNumbersRequest {
}

message GetNumbersResponse {
  int32 value = 1;
}
